lib/Runtime/Language/AsmJsCodeGenerator.cpp

```diff
@@ -37,6 +37,10 @@ namespace Js
 #if ENABLE_DEBUG_CONFIG_OPTIONS
             funcEntrypointInfo->SetIsTJMode(true);
 #endif
+            if (!PreReservedVirtualAllocWrapper::IsInRange((void*)mScriptContext->GetThreadContext()->m_prereservedRegionAddr, (void*)address))
+            {
+                mScriptContext->GetJitFuncRangeCache()->AddFuncRange((void*)address, (int)entrypointInfo->GetCodeSize());
```

Comment:
>(int) [](start = 85, length = 5)

Any issue with typecast to int? GetCodeSize return ptrdiff_t which on x64 is 8 bytes whereas int is just 4?